README.md

```diff
@@ -133,6 +133,44 @@ cd Binaries
 
 Please note the current build script is only tested on Ubuntu 14.04. Please update the script accordingly for other distributions.
 
+## Getting Started on OSX
+
+*Note that OSX is not officially supported yet, but it has been seen to work, YMMV. This build has been tested to work when the dependencies are installed via homebrew, YMMV if using FINK or MacPorts*
+
+Install dependecies with homebrew:
+
+```
+brew install glibmm libxml++ libsigc++ ossp-uuid gettext openssl
+```
```

Comment:
@hanzhumsft it seems the cmake was putting -lintl into the linker commands, which is part of gettext. I'll remove the references to glibmm and libsigc++.